File: Java/Ram.java
public class Ram extends Komponen {

    private int kapasitasGB;
    private String ddr;

    public Ram() {
        super();
        this.kapasitasGB = 0;
        this.ddr = "";
    }

    public Ram(int kapasitasGB, String ddr, String merk, String nama) {
        super(merk, nama);
        this.kapasitasGB = kapasitasGB;
        this.ddr = ddr;
    }

    public void setKapasitasGB(int kapasitasGB) {
        this.kapasitasGB = kapasitasGB;
    }

    public void setDdr(String ddr) {
        this.ddr = ddr;
    }

    public int getKapasitasGB() {
        return kapasitasGB;
    }

    public String getDdr() {
        return ddr;
    }
}

File: Python/Ram.py
from Komponen import Komponen

class Ram(Komponen):
    __kapasitas_gb = 0
    __ddr = ""

    def __init__(self, kapasitas_gb=0, ddr="", merk="", nama=""):
        super().__init__(merk, nama)
        self.__kapasitas_gb = kapasitas_gb
        self.__ddr = ddr

    def set_kapasitas_gb(self, kapasitas_gb):
        self.__kapasitas_gb = kapasitas_gb
    
    def set_ddr(self, ddr):
        self.__ddr = ddr

    def get_kapasitas_gb(self):
        return self.__kapasitas_gb
    
    def get_ddr(self):
        return self.__ddr